DXMainClient/DXGUI/Multiplayer/CnCNet/GameCreationWindow.cs

```diff
@@ -201,8 +200,7 @@ private void BtnLoadMPGame_LeftClick(object sender, EventArgs e)
                 return;
```

Comment:
(could you also please remove the brackets here while you're at it or even merge with the prev. `if`? :P)